flocker/control/_model.py

```diff
@@ -628,6 +628,13 @@ class Deployment(PRecord):
 
     get_node = _get_node(Node)
 
+    @classmethod
+    def create_empty(cls):
+        """
+        Return a ``Deployment`` object representing an empty configuration.
+        """
+        return cls(nodes=frozenset())
```

Comment:
`Deployment.create_empty()` produces the same result as `Deployment()` so I don't think this method provides much value.